homeassistant/components/airvisual/__init__.py

```diff
@@ -97,8 +97,8 @@ def async_get_cloud_coordinators_by_api_key(hass, api_key):
     """Get all DataUpdateCoordinator objects related to a particular API key."""
     coordinators = []
     for entry_id, coordinator in hass.data[DOMAIN][DATA_COORDINATOR].items():
-        config_entry = hass.config_entries.async_get_entry(entry_id)
-        if config_entry.data.get(CONF_API_KEY) == api_key:
+        entry = hass.config_entries.async_get_entry(entry_id)
```

Comment:
Why do we include the clean up to rename config_entry to entry in this PR. It creates loads of unrelated changes that makes it more complicated to review this PR. That's not the purpose of this PR. Please only do one thing per PR.

Reply:
Fair! 👍🏻